configs/nginx/timenow.local

```diff
@@ -0,0 +1,23 @@
+server {
+	listen 443; 
+	listen [::]:443;
+
+	root /var/www/timenow; 
+
```

Comment:
I is also worth to define `index` here pointing to our `index.html` in static root folder.